srcpkgs/python3-hatch/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'python3-hatch'
+pkgname=python3-hatch
+version=1.2.1
+revision=1
+#archs="i686 x86_64"
+wrksrc="${pkgname/python3-/}-${version}"
+#create_wrksrc=yes
+build_style=python3-pep517
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends="python3-hatchling"
+makedepends=""
+depends="python3-atomicwrites python3-click python3-hatchling python3-httpx python3-keyring python3-pexpect python3-platformdirs python3-pyperclip python3-rich python3-tomli-w python3-tomlkit python3-userpath python3-virtualenv"
+short_desc="Modern, extensible Python project management "
+maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
+license="MIT"
+homepage="https://hatch.pypa.io/latest/"
+distfiles="${PYPI_SITE}/h/${pkgname/python3-/}/${pkgname/python3-/}-${version}.tar.gz"
+checksum=61761e1fe14474fb2be7e264555b99fc343b4e63c0ee32aa3497c651e4cdec03
+# Requires itself to be installed for testing
+make_check=no
```

Comment:
It only requires being on import path: `make_check_pre="env PYTHONPATH=./src"`

Reply:
Indeed! With this added (and git added to `checkdepends` it runs mot of the tests now :)